cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -1600,6 +1600,202 @@ More information about workflow endpoint can be found at :ref:`Workflows <http-r
        - | ``[{"runid":"cad83d45-ecfb-4bf8-8cdb-4928a5601b0e","start":1415051892,"end":1415057103,"status":"STOPPED"}]``
 
 
+Retrieving Run Records
+----------------------
+
+To retrieve the latest run records for various applications and different program types, using an HTTP POST method::
+
+  GET /v3/namespaces/<namespace-id>/runs
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``namespace-id``
+     - Namespace ID
+
+with a JSON array in the request body consisting of multiple JSON objects with these parameters:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``appId``
+     - Name of the application being called
+   * - ``programType``
+     - One of ``mapreduce``, ``spark``, ``workflow``, ``flow``, ``service``, or ``worker``
+   * - ``programId``
+     - Name of the program (*mapreduce*, *spark*, *workflow*, *flow*, *service*, or *worker*) being called
+
+The response will be an array of JsonObjects each of which will contain the three input parameters
+as well as 2 fields, "runs" which is a list of the latest run records and "statusCode" which maps to the
+status code for the data in that JsonObjects.
+If an error occurs in the input, then all JsonObjects for which the parameters have a valid status will
+have the count field but all JsonObjects for which the parameters do not have a valid status will have an error
+message and statusCode.
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``runs``
+     - Latest run records for the program defined by the individual JSON object's parameters
+   * - ``statusCode``
+     - The status code from retrieving the instance count of an individual JSON object
+   * - ``error``
+     - If an error, a description of why the status was not retrieved (for example, the
+       specified program was not found, or the requested JSON object was missing a parameter)
+
+**Note:** The ``runs`` field is mutually exclusive of the ``error`` field.
```

Comment:
The runs and error fields are mutually exclusive.

Reply:
I removed this line since it was already in the previous section